### PR TITLE
Searchbar was made

### PR DESCRIPTION
**Pull request by NazariiAlieksieiev**
- [DEMO LINK](https://NazariiAlieksieiev.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://NazariiAlieksieiev.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,45 @@
...
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form
+      class="Big--Searchbar"
+      action="#"
+      method="get"
+      data-qa="big"
+      >
```
Comment (Viktor-Kost) [inline src/index.html:20]:
```suggestion
    <form
      class="Big--Searchbar"
      action="#"
      method="get"
      data-qa="big"
    >
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,45 @@
...
+        src="images/Search.svg" 
+        alt="search" 
+        width="19px"
+      >
+      <input
+      class="Big--Searchbar__input"
+      type="text"
+      placeholder="Try &quot;Los Angeles&quot"
+      data-qa="keypress"
+      >
```
Comment (Viktor-Kost) [inline src/index.html:32]:
```suggestion
      <input
        class="Big--Searchbar__input"
        type="text"
        placeholder="Try &quot;Los Angeles&quot"
        data-qa="keypress"
      >
```

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,45 @@
...
+    </form>
+
+    <form
+      class="Small--Searchbar"
+      action="#"
+      method="get"
+      data-qa="small"
+      >
```
Comment (Viktor-Kost) [inline src/index.html:40]:
```suggestion
    >
```

File: src/index.html [hunk 4]
```diff
@@ -11,6 +11,45 @@
...
+        class="Small--Searchbar__img"
+        src="images/Search.svg" 
+        alt="search"
+        width="11px"
+      >
+      <input 
+      class="Small--Searchbar__input"
+      type="text"
+      placeholder="Try &quot;Los Angeles&quot"
+      >
```
Comment (Viktor-Kost) [inline src/index.html:51]:
```suggestion
      <input 
        class="Small--Searchbar__input"
        type="text"
        placeholder="Try &quot;Los Angeles&quot"
      >
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
+  font-size: 14px;
+  box-sizing: border-box;
+}
+
+::placeholder {
+  font-family: Avenir, Arial, sans-serif;
```
Comment (Viktor-Kost) [inline src/style.css:20]: You don't need to repeat this rule each time, cause you set it at body selector.
If some of tags don't apply it - use `font-family: inherit` instead

Update all over the document

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+input {
```
Comment (Viktor-Kost) [inline src/style.css:23]: Better use class selectors instead of tag ones

Update all over the document

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+input:focus {
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: bold;
+}
+
+form {
+  display: flex;
+  background: #fff;
```
Comment (Viktor-Kost) [inline src/style.css:37]: If you have colors used several times - just make a variable

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+  margin-top: 20px;
+  height: 68px;
+}
+
+.Big--Searchbar__img {
+  margin-left: 26px;
+}
+
+.Big--Searchbar__input {
+  font-size: 16px;
```
Comment (Viktor-Kost) [inline src/style.css:67]: Probably you have to use font-weight for inputs to pass your tests

File: src/style.css (hunk 6 above)
Comment (Viktor-Kost) [inline src/style.css:23]: Reset default padding for inputs

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 9]
```diff
@@ -11,6 +11,45 @@
...
+      method="get"
+      data-qa="big"
+    >
+      <img 
+        class="Big--Searchbar__img"
+        src="images/Search.svg" 
+        alt="search" 
+        width="19px"
+      >
+      <input
```
Comment (liliia-livandovskaya) [inline src/index.html:27]:
```suggestion
      >
      
      <input
```

File: src/index.html [hunk 10]
```diff
@@ -11,6 +11,45 @@
...
+      method="get"
+      data-qa="small"
+    >
+      <img
+        class="Small--Searchbar__img"
+        src="images/Search.svg" 
+        alt="search"
+        width="11px"
+      >
+      <input
```
Comment (liliia-livandovskaya) [inline src/index.html:47]:
```suggestion
      >
      
      <input 
```

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+:root {
+  --grey: #3d4e61;
+  --white: #fff;
+  --creamy: #e1e7ed;
+}
+
+body {
+  font-family: Avenir, sans-serif;
```
Comment (liliia-livandovskaya) [inline src/style.css:20]: always set the font to the `html { ... } tag` and inherit where needed

File: src/index.html [hunk 12]
```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form
+      class="Big--Searchbar"
```
Comment (liliia-livandovskaya) [inline src/index.html:16]: Pay attention to the class names. Fix it all over the page 

Screenshot 2022-11-25 at 11 35 24

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
Pay attention to the comments

File: src/index.html [hunk 13]
```diff
@@ -11,6 +11,45 @@
...
+        src="images/Search.svg" 
+        alt="search" 
+      >
+
+      <input
+        class="input input--big"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot"
+        data-qa="keypress"
+      >
```
Comment (liliia-livandovskaya) [inline src/index.html:32]: you don't need icon--big, form--big, input--big. For the big form, you just need to set the form, icon, and input classes, and change styles according to that. What about small form, you can leave the same classes. The main idea here is not to duplicate code. So you would set styles to the big form, and in the small form you would just set the style that makes them different, you would not need to duplicate all the styles to the small one 

if you don't understand what needs to be changed ask in `fe-chat` and read the checklist one more time

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
Pay attention to the comments

**Review by Viktor-Kost — APPROVED** [review]
Fine

**Pull request closed without merging**